src/pages/de/install/manual.md

```diff
@@ -0,0 +1,143 @@
+
+
+## 1. Erstelle ein Projektverzeichnis
+
+Erstelle ein leeres Verzeichnis mit dem Namen deines Projekts und wechsle dann hinein.
+
+```bash
+mkdir mein-astro-projekt
+cd mein-astro-projekt
+```
+
+Sobald du dich in deinem neuen Verzeichnis befindest, erstelle die Projektdatei `package.json`. Damit verwaltest du die für dein Projekt erforderlichen Pakete, einschließlich Astro. Wenn du mit diesem Dateiformat nicht vertraut bist, führe den folgenden Befehl aus, um die Datei für dich erstellen zu lassen:
+
+```bash
+npm init --yes
+```
+
+
+## 2. Installiere Astro
+
+Füge zuerst Astro zu den erforderlichen Paketen deines Projekts hinzu:
+
+```bash
+npm install astro
+```
+
+Ersetze dann jegliche Platzhalter im Abschnitt `scripts` der Datei `package.json` mit folgenden Astro-Skripts:
+
+```diff
+  "scripts": \{
+-    "test": "echo \"Error: no test specified\" && exit 1"
++    "dev": "astro dev",
++    "start": "astro dev",
++    "build": "astro build",
++    "preview": "astro preview"
+  },
+```
+
+Du kannst diese Skripts später verwenden, um Astro zu starten und seine verschiedenen Befehle auszuführen.
+
+
+## 3. Erstelle deine erste Seite
+
+Verwende deinen Texteditor, um im Projektverzeichnis eine neue Datei mit dem Pfad `src/pages/index.astro` zu erstellen. Sie wird die erste Astro-Seite deines Projekts.
+
+Kopiere im Rahmen dieser Anleitung den folgenden Code-Schnipsel (einschließlich der Bindestriche `---`) und füge ihn in deine neue Datei ein:
+
+```astro
+---
+// Willkommen bei Astro! Alles innerhalb dieses mit drei Bindestrichen
+// vom Rest getrennten Abschnitts nennt man "Komponenten-Frontmatter".
+// Der Code hier wird nie im Browser ausgeführt.
+console.log('This runs in your terminal, not the browser!');
+---
+<!-- Nun folgt deine "Komponenten-Vorlage". Sie besteht aus HTML,
+     ergänzt durch einen Hauch an Magie, um dir beim Erstellen
+     großartiger Vorlagen zu helfen. -->
+<html>
+  <body>
+    <h1>Hallo Welt!</h1>
+  </body>
+</html>
+<style>
+  h1 {
+    color: orange;
+  }
+</style>
+```
+
+
+## 4. Erstelle deine erste statische Ressource
+
+Du solltest auch ein `public/`-Verzeichnis erstellen, um deine statischen Ressourcen zu speichern. Astro kopiert diese Ressourcen beim Buildvorgang immer unverändert in das Ausgabeverzeichnis, sodass du sie sicher aus deinen Komponentenvorlagen heraus referenzieren kannst.
+
+Erstelle in deinem Texteditor eine neue Datei mit dem Pfad `public/robots.txt`. Die Datei `robots.txt` wird von den meisten Websites verwendet, um Suchmaschinen wie Google mitzuteilen, wie sie die Website behandeln sollen.
+
+Kopiere im Rahmen dieser Anleitung den folgenden Code-Schnipsel in die Datei:
+
+```
+# Beispiel: Erlaube allen Suchmaschinen, diese Website zu scannen
+# und in den Suchindex aufzunehmen.
+# Volle Syntax: https://developers.google.com/search/docs/advanced/robots/create-robots-txt
```

Comment:
I think you meant "Voller Syntax" instead of "Volle Syntax" (forgot the r in "Voller")

Reply:
Nein, "Syntax" ist tatsächlich weiblich. https://www.duden.de/rechtschreibung/Syntax

Reply:
Oh ok :)